src/modules/Lan.rb

```diff
@@ -713,13 +725,13 @@ def Import(settings)
     def Export
       profile = Y2Network::AutoinstProfile::NetworkingSection.new_from_network(yast_config)
       ay = {
-        "dns"                  => profile.dns ? profile.dns.to_hashes : {},
-        "net-udev"             => profile.udev_rules ? profile.udev_rules.udev_rules.map(&:to_hashes) : [],
-        "s390-devices"         => profile.s390_devices ? profile.s390_devices.to_hashes["devices"] : [],
+        "dns"                  => profile.dns&.to_hashes || {},
+        "net-udev"             => profile.udev_rules.udev_rules&.map(&:to_hashes) || [],
```

Comment:
```suggestion
        "net-udev"             => profile.udev_rules&.udev_rules.map(&:to_hashes) || [],
```

Reply:
no, this change will crash. as `&.` can return nil and in such case `nil.map` crash.

Reply:
I think RuboCop has a bug in not distinguishing one `udev_rules` from the other. I cannot really blame it, such API is crazy.
Still, your change is not equivalent. You may want to use TWO `&.` then.

Reply:
ah, right, two `&.` looks correct for me.